### PR TITLE
backend: transition to @safe code part 5

### PR DESCRIPTION
**Pull request by WalterBright**
See the rationale here #12327

### CODE REVIEW

File: src/dmd/backend/dwarf.d [hunk 1]
```diff
@@ -18,6 +18,7 @@ import dmd.backend.type;
 extern (C++):
 
 nothrow:
+@safe:
```
Comment (Geod24) [inline src/dmd/backend/dwarf.d:21]: Same here, those are trusted

Comment (WalterBright) [reply src/dmd/backend/dwarf.d:21]: same answer, buildkite dstyle rejects `@trusted:` and `@trusted { ... }`

**Comment by Geod24** [comment]
@WalterBright : You could just opt-out of the `@trusted` check, see https://github.com/dlang/dmd/blob/be9dc9d1c310a184406584d1036c2ef9884a7c06/src/.dscanner.ini

**Comment by WalterBright** [comment]
I'm not going to futz around with dscanner. I don't think configuring dscanner differently based on which model is a proper use of it.

It doesn't matter for the purpose of dmd if those declarations are marked as @safe. If you still strongly object, I recommend changing dscanner's default behavior, not special casing it.

**Review by RazvanN7 — APPROVED** [review]

**Pull request merged**